build.rs

```diff
@@ -491,6 +493,17 @@ fn build_c_code(
     );
 }
 
+fn cc_builder() -> cc::Build {
```

Comment:
Is there a more generic convention for `--sysroot` that isn't WASI-specific? Presumably `TARGET_CFLAGS=--sysroot=${WASI_SDK_DIR}` works fine?